src/rust/src/types.rs

```diff
@@ -28,6 +28,21 @@ impl LazyPyImport {
             })
             .map(|p| p.as_ref(py))
     }
+
+    pub fn get_bound<'p>(
```

Comment:
`self.get()` should just call this I think.

Reply:
I did this via `Ok(self.get_bound(py)?.into_gil_ref())`, not sure if that's optimal :slightly_smiling_face: